recipes/xvfbwrapper/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set version = "0.2.8" %}
+
+package:
+  name: xvfbwrapper
+  version: {{ version }}
+
+source:
+  fn: xvfbwrapper-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/f0/06/8462744fbdb980cd6ffaee23d98ef6edc15a13428a85ac7ba10dd6b5989d/xvfbwrapper-{{ version }}.tar.gz
```

Comment:
That's because it has the package name another time. So it looks like this.

```
https://pypi.io/packages/source/x/xvfbwrapper/xvfbwrapper-0.2.8.tar.gz
```